src/main/java/org/folio/rest/transaction/AllOrNothingHandler.java

```diff
@@ -384,7 +384,8 @@ protected String getValues(List<JsonObject> entities) {
 
   private Future<Void> checkEncumbranceRestrictions(Transaction transaction, Budget budget) {
```

Comment:
Please give this method a more general name, as it checks not only encumbrances, but also payments.

Reply:
Sure. Naming updated